### ISSUE TITLE
Question title styling modifications

### ISSUE THREAD

**Issue by boonebgorges** [livinglab/webwork-for-wordpress#149]
On the Question List page, question titles should be underlined, to indicate that they're links. Remove underline on hover for a11y.

Title should be black and NOT underlined on single problem page, according to https://www.dropbox.com/s/enr47xuwh1rsico/Screenshot%202019-03-21%2015.32.04.png?dl=0. @jennaspevack @bree-z Note that the question title on single problem pages is a link, primarily so that you can easily get a permalink to the specific question (right-click, "copy link location"). Should I be removing the link, in addition to removing the link styling?

**Comment by jennaspevack** [comment]
My thought was that underlined text indicates a link to "somewhere" and could be confusing to users.  Rather, could we add the permalink below the header, next to the Edit button? @boonebgorges is it easy to add a copy permalink function? See mockup. Alternatively, we could replace the COPY text with PERMALINK. @drdrew42 does having the permalink accessible in this way seem like it would be useful to students/faculty?
![image](https://user-images.githubusercontent.com/22797200/56395508-e1427480-6208-11e9-8946-d0ce08dddb1b.png)

**Comment by boonebgorges** [comment]
The idea that the title should be a permalink comes from me - it's a convention around the web for this kind of item that either the title itself, or the timestamp ("Posted March 14, 2019") is a permalink to the current item. If the team doesn't think it's useful, we can take it out; and if it's not useful in its current iteration, then it seems unlikely to be worth adding another bit of UI (the "Permalink" or "Copy link" button) just for this purpose. My point in asking the question here was just that removing the link styling, without also removing the link, creates some UI inconsistency. Happy to do whatever the team thinks is best.

**Comment by jennaspevack** [comment]
Ah, I see. I guess if the timestamp is also a convention, I'd put it there instead to avoid confusion. Or just remove it altogether, if moving it to the timestamp is out of scope.

**Comment by boonebgorges** [comment]
Thanks, Jenna! I've made the following changes:

a. Timestamps "Posted ..." are now links, both on the List and Single views. I've allowed the theme's link styling to bleed through, so that this text is now blue instead of gray (but no underlines)
b. On the List view, titles are now underlines to indicate that they're links
c. On the Single view, titles are no longer links

Screenshots:
![Screenshot_2019-04-29_13-49-45](https://user-images.githubusercontent.com/246627/56919295-d946c800-6a85-11e9-96bb-c0b7d72a6119.png)
![Screenshot_2019-04-29_13-49-26](https://user-images.githubusercontent.com/246627/56919296-d946c800-6a85-11e9-98cf-c48417c4f376.png)

@jennaspevack Please let me know if I've captured the spirit, or if you'd like changes.

**Comment by jennaspevack** [comment]
Boone, quick question. 

Could we get a hover for the title? Perhaps _black, no underline_ to keep with the rest of the links and also to differentiate the visited link?

Sorry, I didn't realize the hover was missing on the live site...

**Comment by boonebgorges** [comment]
I've added a declaration that removes the underline on hover. The :visited styling isn't working consistently for me across browsers, but I'm afraid that this may be a browser security limitation that I can't fix. Let me know what you think.

**Comment by bree-z** [comment]
This looks good to me!  I do also see inconsistencies across browsers with the visited link color.  But, everything else is working as expected.  

@jennaspevack I'm not sure if you want to take a look.  Otherwise we can close this one.  Thanks!

**Comment by jennaspevack** [comment]
Thanks @boonebgorges! The hover/visited looks good to me, but noticing that the anonymous link and regular hover styles are strangely overlaid. It's not clear that the anonymous link is different from a regular link because the underline and dotted line are layered.
Current anonymous link: 
Current visited anonymous link:

Is it terribly complicated to remove the regular underlined link and just have the dotted line for anonymous question titles? And make the dotted underline black for visited (though maybe this whole visited business isn't worth the effort due to browser inconsistencies.)

Anonymous link: https://www.dropbox.com/s/wyyr4syi7m6nsvh/anonomous-link.png?dl=0
Anonymous visited link: https://www.dropbox.com/s/2mlfmy2ljakdn0e/anonomous-link-visited.png?dl=0

**Comment by boonebgorges** [comment]
I think I've captured your feedback, @jennaspevack - see openlabdev.org or the gif below, where I look at both a visited and an unvisited anonymous question link.

![Peek 2019-06-19 20-35](https://user-images.githubusercontent.com/246627/59811999-dfad2f80-92d1-11e9-99fa-a3fa05de8d5f.gif)

**Comment by jennaspevack** [comment]
Looks good for logged in view! 

Oy, however logged out view doesn't have the underline link styling for anonymous. For the logged out view can we get an underline to keep the link style consistent with the other links?

See here: https://www.dropbox.com/s/1uiefgsq2cu7syv/loggedoutview-anonomouslink.mov?dl=0

**Comment by boonebgorges** [comment]
Thanks, Jenna. I think I've made the proper change, but the logic is getting pretty tangled, so another test from you would be helpful.

**Comment by jennaspevack** [comment]
Phew. Perfect. Thanks!
